docs/release_notes/5.0.15-13.md

```diff
@@ -0,0 +1,43 @@
+
+It supports MongoDB 5.0.14 protocols and drivers 
+
+## Release Highlights
+
+* The support for authentication using [AWS IAM](../aws-iam.md) enables you to natively integrate Percona Server for MongoDB with AWS services, increase security of your infrastructure by setting up password-less authentication and offload your DBAs from managing different sets of secrets. This is the [technical preview feature](../glossary.md#technical-preview-feature)
+* Improved master key rotation for data at rest encrypted with HashiCorp Vault enables you to use the same secret key path on every server in your entire deployment thus significantly simplifying the secrets management and key rotation process.
+
+The bug fixes, provided by MongoDB and included in Percona Server for MongoDB, are the following:
+
+* [SERVER-54900](https://jira.mongodb.org/browse/SERVER-54900) - Fixed the issue with blocked networking calls preventing the oplog fetching process to re-establish the connection to an unresponsive sync-source node 
+* [SERVER-72416](https://jira.mongodb.org/browse/SERVER-72416) - Fixed the issue with incorrect projection parsing when a collection level collation is specified
+* [SERVER-71759](https://jira.mongodb.org/browse/SERVER-71759) - Changed the yielding policy of dataSize command to YIELD_AUTO for both when the command is called with estimate:true or false
+* [SERVER-72535](https://jira.mongodb.org/browse/SERVER-72535) - Disallow creating the ‘admin’, ‘local’, and ‘config’ databases with alternative cases in names on sharded clusters
+* [SERVER-72222](https://jira.mongodb.org/browse/SERVER-72222) - Fixed the incorrect behavior of the `mapReduce` command with single reduce optimization in sharded clusters
+
+Find the full list of changes in the [MongoDB 5.0.15 Community Edition release notes](https://www.mongodb.com/docs/manual/release-notes/5.0/#5.0.15---feb-27--2023).
+
+
+## New Features
+
+* [PSMDB-1033](https://jira.percona.com/browse/PSMDB-1033): Add authentication with AWS IAM
+
+## Improvements
+
+* [PSMDB-1148](https://jira.percona.com/browse/PSMDB-1148): Improve the master key rotation when using a single master key for data-at-rest encryption with Vault in the entire deployment
+
+
+## Bugs Fixed
+
+* [PSMDB-1201](https://jira.percona.com/browse/PSMDB-1201): Improved the error message if the attempt to save an encryption key to a KMIP server failed
+* [PSMDB-1203](https://jira.percona.com/browse/PSMDB-1203) - Gracefully terminate mongod if the master encryption key can’t be saved to a KMIP server
+* [PSMDB-1204](https://jira.percona.com/browse/PSMDB-1204) - Fixed the handling of attributes list for LDAP authentication with OpenLDAP during the user to DN mapping stage
+* [PSMDB-1229](https://jira.percona.com/browse/PSMDB-1229): Fixed the issue with PSMDB exiting after restart by properly reading of the master key identifier
```

Comment:
It seems there is a formatting inconsistency in the `Bugs Fixed` list. The first and the fourth line have the following format:
```
[<ticket_id>](URL): <description>
```
but the second and the third ones have ` -` (please note the space before the dash character) instead of `:`.

Moreover, the same inconsistency seems to exist in the broader scope. In the `Release Highlights` section,  each line uses ` -` whilst the `New Features` and `Improvements` rely on `:`. Finally, there is a mix of two styles in `Bugs Fixed`. Is there some rationale behind that?

Reply:
Fixed styling in Bugs Fixed